docs/developer.rst

```diff
@@ -85,6 +85,8 @@ for MySQL.
 Create if not exists, and add/append the configuration details as below.
 PostgresSQL conf file `~/.pgpass` file.
 
+For more information regarding Passwordless configuration you can visit `Password File`_  
```

Comment:
I would also recommend we add the link to MySQL Passwordless configuration
For more information regarding Passwordless configuration you can visit `PostgresSQL Password File`_ and `MySQL Password File`